doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3550,6 +3550,11 @@ Trusted Firmware-M (TF-M)
 
 The issues in this section are related to the TF-M implementation in the |NCS|.
 
+.. rst-class:: v2-4-0
+
+NCSDK-22818: TF-M does not build with :kconfig:option:`CONFIG_TFM_PROFILE_TYPE_MINIMAL` disabled and :kconfig:option:`CONFIG_TFM_LOG_LEVEL_SILENCE` enabled.
+  This will fail with the error: message "undefined reference to 'end'" caused by printf being included in the build
```

Comment:
```suggestion
  This fails with the error: message "undefined reference to 'end'" caused by printf being included in the build.
```

Can we change those single quotation marks ' ' to backticks ` ` ? Looks a bit odd as is.

Reply:
Not sure that is wise, backticks are intepreted differently and could have an affect on the doc build.